lib/ProcessContainer.js

```diff
@@ -207,7 +207,7 @@ function exec(script, stds) {
         if (pm2_env.pm_err_log_path !== 'NULL' && pm2_env.pm_err_log_path !== '/dev/null' && stds.err.write) {
           stds.err.write(log_data, encoding, cb);
         }
-        else {
+        else if (cb) {
```

Comment:
Sure, I can change it, if you prefer that version. But the whole codebase seems to use boolean coercion all the time – just as I did.

Some examples:

```JavaScript
if (process.connected &&
    process.send &&
    process.versions &&
    process.versions.node)
        …
  });
```

or 

```JavaScript
if (… && stds.out.write) {
    …
}
```